docs/guides/docs/intro-to-sway/checkpoint.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Checkpoint
+category: Intro to Sway
+parent:
+  label: Guides
```

Comment:
Can you change all of the parent labels to "All Guides" to match the other guides?